mh/pat_item.py

```diff
@@ -45,7 +45,7 @@ def lp_string(s):
 def unpack_lp_string(data, offset=0):
     """Unpack lp_string."""
     size, = struct.unpack_from(">B", data, offset)
-    return data[offset+1:offset+1+size]
+    return data[offset + 1:offset + 1 + size]
```

Comment:
I'm not sure which linter you're using but AFAIK it's not a PEP8 violation and can be left as is. I don't mind the space, though, so feel free to keep them.